runtime/data/test_data_with_header_and_footer.csv

```diff
@@ -0,0 +1,11 @@
+header
```

Comment:
If there is a different number of fields than 4, get_data() skips it. The comment said it was for checking for headers and footers, so I've added a file with data for testing this piece of code. I didn't want to mess with test_data.csv